tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,98 @@
+from tensorflow.python.ops import array_ops
+from tensorflow.python.util.tf_export import tf_export
+from tensorflow.python.ops import math_ops
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import image_ops_impl
+from tensorflow.python.ops import script_ops
+from tensorflow.python.ops import control_flow_ops
+from tensorflow.python.ops import gen_math_ops
+from tensorflow.python.framework import tensor_shape
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.ops import gen_array_ops
+
+
+
+@tf_export('image.median_filter_2D')
```

Comment:
No need to use `tf_export` anymore. We expose the methods using the [init file](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/image/__init__.py)

Reply:
solved